src/status_im2/contexts/profile/data/events.cljs

```diff
@@ -0,0 +1,157 @@
+(ns status-im2.contexts.profile.data.events
+  (:require [clojure.string :as string]
+            [native-module.core :as native-module]
+            [quo2.foundations.colors :as colors]
+            [re-frame.core :as re-frame]
+            [react-native.platform :as platform]
+            [status-im.multiaccounts.update.core :as multiaccounts.update]
+            [status-im2.common.theme.core :as theme]
+            [status-im2.constants :as constants]
+            [status-im2.contexts.shell.jump-to.utils :as shell.utils]
+            [status-im2.setup.hot-reload :as hot-reload]
+            [taoensso.timbre :as log]
+            [utils.re-frame :as rf]))
+
+;; TODO(alwx): this function must be removed once the old bottom sheet is completely removed
```

Comment:
just wondering how bottom sheet is related to profile? why is it here ?

Reply:
Removed